velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1528,6 +1528,25 @@ struct CurrentDateFunction {
   }
 };
 
+template <typename T>
+struct CurrentTimestampFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  int64_t packed_{0};
+
+  FOLLY_ALWAYS_INLINE void initialize(
+      const std::vector<TypePtr>&,
```

Comment:
Typically we have a variable name which is commented for an unused parameter

`const std::vector<TypePtr>& /* type */,`

Reply:
Done.